javascript/builtins/Promise.json

```
@@ -584,7 +584,14 @@
               },
               "edge": "mirror",
               "firefox": {
-                "version_added": false
+                "version_added": "132",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "javascript.options.experimental.promise_try",
+                    "value_to_set": "true"
+                  }
+                ]
```

@chrisdavidmills If this is in nightly, it gets done like this

```suggestion
                "version_added": "preview"
```

---

Note, flag disappears.

---

Ah, so "preview" is the word I was looking for. Thanks @hamishwillee!

Q: Why get rid of the flag? Surely that's important information that folks wanting to test the Fx implementation will want to know?

---

Well you can just test on the release to find out the "non implemented version" and nightly to test the change.
Personally I am fine with the flag being there, and even see it as beneficial, but it isn't "needed".
Either way, "that's the BCD way"